src/appleseed/foundation/utility/stopwatch.h

```diff
@@ -83,6 +89,8 @@ class Stopwatch
     uint64  m_overhead;     // measured overhead of calling start() + measure()
     uint64  m_start;        // timer value when start() is called
     uint64  m_elapsed;      // elapsed time when measure() is called, adjusted for overhead
+    uint64  m_stock;        // elapsed time until pause() is called, adjusted for overhead
```

Comment:
What do you mean (in French maybe) by `stock`?

Reply:
All the time spent before pressing calling `pause`

Reply:
Ok, we'll need to find a better, more explicit name. I could help with naming but I need to fully understand how it works.

Can we call `pause()` multiple times?

Reply:
Now we can't, I added an assert. But once you resume, you can call `pause` again

Reply:
Ok, that's what I meant. It's fine to assert if `pause()` is called while already in pause, but of course we should allow calling `pause()` and `resume()` multiple times. If that's what we do then great.

So, here's an idea:

1. In `measure()`, you we _accumulate_ (add) elapsed time to `m_elapsed`.
2. We move the overhead subtraction to `get_ticks()`.
3. We get rid of `m_stock`.

Reply:
Thanks for the idea, I haven't tried to use the overhead in `get_ticks` which is why I had problems.

Reply:
The only issue with that, is that we won't be able to call `measure` if the timer is paused. Otherwise `ticks` would be wrong

Reply:
Unless `measure()` doesn't do anything if paused?

Reply:
That's what I implemented :) pushing tomorrow